<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TianLiQtCommon_SelectedItemButton</class>
 <widget class="QPushButton" name="TianLiQtCommon_SelectedItemButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>27</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>218</width>
    <height>27</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>218</width>
    <height>27</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TianLiQtCommon_SelectedItemButton</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.488636, radius:1.502, fx:0.318, fy:0.227273, stop:0 rgba(235,233,227, 180), stop:0.659091 rgba(228,221,213, 180), stop:1 rgba(224,215,206, 180));
border-radius:5px;
border:none;
}
QPushButton:hover{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.488636, radius:1.502, fx:0.318, fy:0.227273, stop:0 rgba(235,233,227, 255), stop:0.659091 rgba(228,221,213, 255), stop:1 rgba(224,215,206, 255));
}
QPushButton:checked{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.488636, radius:1.502, fx:0.318, fy:0.227273, stop:0 rgba(235,233,227, 255), stop:0.659091 rgba(228,221,213, 255), stop:1 rgba(224,215,206, 255));
}</string>
  </property>
  <property name="checkable">
   <bool>true</bool>
  </property>
  <property name="autoExclusive">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label_Image">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>-10</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>0</y>
     <width>130</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HYWenHei</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-size:12px;</string>
   </property>
   <property name="text">
    <string>空物品</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Region">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>6</y>
     <width>50</width>
     <height>10</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HYWenHei</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel{
font-size:8px;
color:rgba(0,0,0,128);
}</string>
   </property>
   <property name="text">
    <string>地点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_LabelImage">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>-9</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Progress_Bg">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>18</y>
     <width>85</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel{
background-color:rgba(147,95,30,26);
border-radius:1px 0px 0px 1px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Progress_A">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>18</y>
     <width>20</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel{
background-color:rgba(55,255,255,255);
border-radius:1px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Progress_B">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>18</y>
     <width>20</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel{
background-color:rgba(255,255,255,1);
border-radius:1px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_LabelImage</zorder>
  <zorder>label_Image</zorder>
  <zorder>label_Title</zorder>
  <zorder>label_Region</zorder>
  <zorder>label_Progress_Bg</zorder>
  <zorder>label_Progress_B</zorder>
  <zorder>label_Progress_A</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TianLiQtCommon_SelectItemButton.qrc"/>
 </resources>
 <connections/>
</ui>
